pkgs/development/python-modules/pyenchant/default.nix

```
@@ -13,25 +13,26 @@ buildPythonPackage rec {
     sha256 = "fc31cda72ace001da8fe5d42f11c26e514a91fa8c70468739216ddd8de64e2a0";
   };
 
-  propagatedBuildInputs = [ pkgs.enchant1 ];
-
-  patchPhase = let
-    path_hack_script = "s|LoadLibrary(e_path)|LoadLibrary('${pkgs.enchant1}/lib/' + e_path)|";
-  in ''
-    sed -i "${path_hack_script}" enchant/_enchant.py
+  propagatedBuildInputs = [ pkgs.enchant2 ];
```

pkgs shouldn't be passed directly, please do 
```suggestion
  propagatedBuildInputs = [ enchant2 ];
```

and change the calling location in python-packages.nix to:
```
  pyenchant = callPackage ... {
    enchant2 = pkgs.enchant2;
  };
```

---

Done in the latest commit; should I squash these into a single commit?

---

yes, please squash

---

```
git reset HEAD^^
git add pkgs/
git commit --amend --no-edit
```

---

Done in latest